packages/tags/src/Tag.ts

```diff
@@ -70,17 +70,47 @@ export class Tag extends SizedMixin(SpectrumElement, {
         this.removeEventListener('focusout', this.handleFocusout);
     };
 
+    /**
+     * This function, named 'focusAfterDelete', is used to handle the focus behavior after
+     * deleting an element within a parent element.
+     * @param parent
+     */
+    private focusAfterDelete = (parent: HTMLElement): void => {
```

Comment:
This feels pretty similar to what we might need in https://github.com/adobe/spectrum-web-components/issues/3091. Could we delegate this work to Focus Group rather than building it into Tag?

Reply:
Make sense.